cli/index.ts

```diff
@@ -31,6 +28,19 @@ async function main() {
 				return value;
 			},
 		} )
+		.option( 'avoid-telemetry', {
+			type: 'boolean',
+			hidden: true,
+		} )
+		.middleware( async ( argv ) => {
+			if ( ! argv.avoidTelemetry ) {
```

Comment:
Do we need this condition? AFAIU `avoidTelemetry` is always `true`? Or I misunderstand something?

Reply:
It's not true when the CLI is initiated directly from the terminal